app/views/layouts/_application_base.html.erb

```diff
@@ -8,6 +8,28 @@
     <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
     <meta name="turbo-cache-control" content="no-cache">
 
+    <script>
+    (function (m, a, z, e) {
+      var s, t;
+      try {
+        t = m.sessionStorage.getItem('maze-us');
+      } catch (err) {}
+
+      if (!t) {
+        t = new Date().getTime();
+        try {
+          m.sessionStorage.setItem('maze-us', t);
+        } catch (err) {}
+      }
+
+      s = a.createElement('script');
+      s.src = z + '?apiKey=' + e;
+      s.async = true;
+      a.getElementsByTagName('head')[0].appendChild(s);
+      m.mazeUniversalSnippetApiKey = e;
+    })(window, document, 'https://snippet.maze.co/maze-universal-loader.js', 'dfa66380-6f7f-4d3b-99f9-439b5226c0c2');
```

Comment:
Si je comprends bien le snippet c'est la clé d'API que tu mets en clair non 😬 ? 
Je pense que même si c'est temporaire il aurait fallu faire quelque chose de similaire à ce qu'on fait avec [matomo](https://github.com/gip-inclusion/rdv-insertion/blob/staging/app/javascript/components/matomo-script-tag.js)

Reply:
Oui effectivement, mais cette clé n'a pas vocation à être privée, elle sera de toute façon en clair dans le DOM in fine. Mais oui on peut la mettre en var d'env